src/Acuminator/Acuminator.Tests/Tests/StaticAnalysis/LongOperationStart/LongOperationInPXGraphDuringInitializationTests.cs

```diff
@@ -5,13 +5,14 @@
 using Acuminator.Tests.Verification;
 using Microsoft.CodeAnalysis.Diagnostics;
 using Xunit;
+using Acuminator.Utilities;
 
 namespace Acuminator.Tests.Tests.StaticAnalysis.LongOperationStart
 {
     public class LongOperationInPXGraphDuringInitializationTests : DiagnosticVerifier
     {
         protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer () =>
-            new PXGraphAnalyzer(new LongOperationInPXGraphDuringInitializationAnalyzer());
+            new PXGraphAnalyzer(CodeAnalysisSettings.Default, new LongOperationInPXGraphDuringInitializationAnalyzer());
```

Comment:
Don't you need `WithRecursiveAnalysisEnabled()` here? Tests should not rely on the default settings because they can change in the future; all settings that are used in the tests must be set explicitly.

Reply:
Ok